### ISSUE TITLE
新的术语

### ISSUE THREAD

**Issue by MasterGodzilla** [jiqizhixin/Artificial-Intelligence-Terminology#30]
这个东西也太好了！不过最近大模型出来，可能需要多加一点术语。

----------计算机视觉相关----------
Diffusion Model 扩散模型
Consistency Model 一致性模型（？）
Contrastive Learning 对比学习
Flow Matching 流匹配/流对齐（？）
Rectified Flow 修正流（？）

------大语言模型相关-------
Alignment 对齐
Intruction tuning 指令微调
LORA 低秩微调（？）
pretraining/post-training 预训练/后训练
Preference 偏好

-----------其他-----------
Gumbel Distribution 耿贝尔分布（？）ref （https://zh.wikipedia.org/wiki/%E8%80%BF%E8%B4%9D%E5%B0%94%E5%88%86%E5%B8%83）
Gumbel softmax/gumbel trick/Gumbel sampling 耿贝尔采样(?)
curriculum learning 课程训练？
Retrivial？Retrieval Augmented Generation?
